dataservice/api/biospecimen/schemas.py

```diff
@@ -64,14 +64,21 @@ class Meta(BaseSchema.Meta):
         'biospecimen_diagnoses': ma.URLFor(
             'api.biospecimen_diagnoses_list', biospecimen_id='<kf_id>'),
         'diagnoses': ma.URLFor('api.diagnoses_list',
-                               biospecimen_id='<kf_id>')
+                               biospecimen_id='<kf_id>'),
+        'genomic_files': ma.URLFor('api.genomic_files_list',
+                                   biospecimen_id='<kf_id>')
     })
 
 
 class BiospecimenFilterSchema(BiospecimenSchema):
 
     diagnosis_id = fields.Str()
+    genomic_file_id = fields.Str()
```

Comment:
What does this do? I don't think the biospecimen has a single `genomic_file_id`, does it?

Reply:
This field captures what we specify while filtering in the query. ya biospecimes might not have single genomic_file and vice versa. I think we cannot provide `_links` for this reason.

Reply:
Hmm @parimalak @dankolbman ok so do you think that _links for a resource should purely reflect the relations of the underlying database entities? Idk if I like that.. Should links express how you can go from one resource to other resource(s) that are related to it. A biospecimen IS related to a genomic file, through a many to many relationship, which is implemented with the intermediate association table. So I think having a link for biospecimens, that returns biospecimens related to a genomic file is valid (and vice versa).